grakn-graql/src/test/java/ai/grakn/graql/internal/query/AdminTest.java

```diff
@@ -105,12 +105,18 @@ public void testMutateMatch() {
     public void testInsertQueryMatchPatternEmpty() {
         InsertQuery query = qb.insert(var().id(ConceptId.of("123")).isa("movie"));
         assertFalse(query.admin().match().isPresent());
+
+        query = qb.insert(var().id(ConceptId.of("123")).directIsa("movie"));
+        assertFalse(query.admin().match().isPresent());
```

Comment:
Did something go wrong that meant you added this? If not, I sort of think we can remove these new tests.